mailbox/plugin/quota-search-elasticsearch/src/main/java/org/apache/james/quota/search/elasticsearch/events/ElasticSearchQuotaMailboxListener.java

```diff
@@ -64,17 +67,21 @@ public boolean isHandling(Event event) {
     }
 
     @Override
-    public void event(Event event) throws IOException {
-        handleEvent((QuotaUsageUpdatedEvent) event);
+    public Publisher<Void> reactiveEvent(Event event) {
```

Comment:
we are not in api right? I think can return a Mono then if you want. But I don't see any issues with a Publisher either, so it's just a remark :)

Reply:
There's no direct calls, hence this makes no difference.